sql/modules/Roles.sql

```diff
@@ -415,7 +539,14 @@ SELECT lsmb__grant_menu('employees_manage', 49, 'allow');
 GRANT select ON employees TO public;
 
 
-SELECT lsmb__create_role('contact_edit');
+SELECT lsmb__create_role('contact_edit',
+                         $DOC$
+                         This role allows editing of existing contacts.
+
+                         Combine this role with one or more 'contact_class_*' roles to be able to access
+                         contacts of the specific class. By itself, this role does not provide sufficient rights.
+                         $DOC$
+);
```

Comment:
I have the same question as for `contact_read` and `contact_create`, what does it do on its own, but I suspect it doesn't do anything?

Reply:
This role can change existing contacts (which inherently means being able to read their data). However, with this role you can't create new ones.

Reply:
maybe it should say something like:

> allows editing of existing contacts, but not creation of new contacts.

just to be explicit?